oauthdoctor/diag/config_test.go

```diff
@@ -252,6 +255,55 @@ func TestPrint(t *testing.T) {
 	}
 }
 
+func TestReplaceConfig(t *testing.T) {
+	log.SetOutput(ioutil.Discard)
+
+	now := time.Now().Format("2006-01-02_")
+	dir, err := os.Getwd()
+	if err != nil {
+		t.Errorf("Error getting current dir: %s", err)
+	}
+
+	tests := []struct {
+		desc   string
+		cfg    ConfigFile
+		backup string
+	}{
+		{
+			desc: "Config file and backup file exists",
+			cfg: ConfigFile{
+				Filepath: filepath.Join(dir, "testdata"),
+				Filename: "python_config2",
+				Lang:     "python",
+			},
+			backup: "diag/testdata/python_config2_" + now,
+		},
+	}
+
+	for _, test := range tests {
+		backup := test.cfg.ReplaceConfig(DevToken, "randomToken")
+
+		config := test.cfg.GetFilepath()
+		_, err := os.Stat(config)
+		if os.IsNotExist(err) {
```

Comment:
Is this the only error that can be produced by os.Stat?

Reply:
In godoc, it says `If there is an error, it will be of type *PathError.`

https://golang.org/pkg/os/#Stat

PathError can be more than a missing file. I added else if to catch all other potential errors.